templates/base.html.twig

```diff
@@ -45,12 +45,12 @@
         </div>
         {% if app.user %}
             <div class="clearfix page-header-user">
-                <form method="post" action="{{ logout_url('saml_based') }}" class="pull-right">
+                <a href="{{ logout_url('saml_based') }}" class="pull-right">
```

Comment:
I don't think this will work because it changes the POST to a GET, and selfservice does not allow GET requests to the logout endpoint.

Reply:
That's cheating, because the GET-method was only removed two weeks ago ;)

Reply:
Not quite, in 4.0.9 (released Jun 6, 2023) I can also not use GET for logout, it only accepts POST.